handlers/check.go

```diff
@@ -32,8 +32,7 @@ func CheckHandler(w http.ResponseWriter, r *http.Request) {
 	repo, err := download.Clean(r.FormValue("repo"))
 	if err != nil {
 		log.Println("ERROR: from download.Clean:", err)
-		w.WriteHeader(http.StatusBadRequest)
-		w.Write([]byte(`Could not download the repository: ` + err.Error()))
+		http.Error(w, `Could not download the repository: `+err.Error(), http.StatusBadRequest)
```

Comment:
@tariq1890 can we make these "  rather than ` too? I don't think there was any reason to use ` here before